phone/src/os/call/hooks/useCallService.tsx

```diff
@@ -57,7 +55,6 @@ export const useCallService = () => {
   }, [history, modal, pathname, modalHasBeenOpenedThisCall]);
 
   const _setCall = (_call: CallProps) => {
-    resetDuration();
```

Comment:
Wasn't an issue when I tested, as I tweaked the timer to mount/unmount depending on whether the call is accepted or not. Double checking this with another person would help test that though.

Reply:
Yea in the past it would store your call state from your last call. 

You can test this by yourself by calling yourself > answering > letting the duration run > end > and retry. 

The bug was reproduced in this environment previously so you can most likely reproduce it this way again, should it exist.

Reply:
Then its resolved